concent_api/core/message_handlers.py

```diff
@@ -1301,7 +1309,7 @@ def handle_send_subtask_results_verify(
             reason=message.concents.ServiceRefused.REASON.InvalidRequest,
         )
 
-    if Subtask.objects.filter(
+    if Subtask.objects.filter(  # pylint: disable=no-member
```

Comment:
Having pylint complain about `no-member` on a simple `filter()` call is bizarre. This should not be necessary. There is something really weird going on here. Please investigate.